docs/openhabian-amanda.md

```diff
@@ -111,7 +111,7 @@ That is, you have to mount the USB stick or disk from your NAS to a directory th
 Specifically for Windows users: if you are not familiar with the UNIX filesystem concept and what it means 'to mount' storage,
 read up on it NOW. Various tutorial can be found on the net such as https://linoxide.com/linux-how-to/how-to-mount-drive-in-linux .
 Google is your friend, but it'll give a lot of answers, each to vary slightly depending on the Linux variant or use case.
-Make sure you ask specific questions such as “how to mount a NAS disk on a raspbian raspberry pi”.
+Make sure you ask specific questions such as “how to mount a NAS disk on a Raspberry Pi OS raspberry pi”.
```

Comment:
I don't think that'll give many hits ;-)

Reply:
What about now?

Reply:
> What about now?

ok